lib/bridge/MatrixHandler.js

```diff
@@ -369,7 +368,90 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
         return;
     }
     else {
-        req.log.info("No valid admin command: %s", event.content.body);
+        let fmt = '\nirc.server\n'+
+                    '(COMMAND [arg0 [arg1 [...]]])\n'+
+                    '(COMMAND [arg0 [arg1 [...]]])\n'+
+                    'irc.server2';
```

Comment:
I find this has the tendency to insert the indentations within the file as well, am I doing something wrong?